.rubocop.yml

```diff
@@ -11,3 +11,7 @@ Metrics/BlockLength:
     - 'features/**/*.rb'
     - 'spec/**/*'
     - 'config/routes.rb'
+
+inherit_gem:
```

Comment:
I wonder if all the additional linting errors would disappear if we moved this to the top of the file?

Reply:
@benthorner it works now that I've excluded temp files.